README.md

```diff
@@ -57,6 +57,10 @@ JSON file. You may also set the GOOGLE_APPLICATION_CREDENTIALS environment varia
     ./cloud_sql_proxy -dir=/cloudsql -instances=my-project:us-central1:sql-inst=tcp:3306 &
     mysql -u root -h 127.0.0.1
 
+    # For specifying another host:
```

Comment:
Can you add a warning here that care should be taken when running this command? If executing on a system without a correctly configured firewall this will could potentially allow anything on the internet to access the database.